### ISSUE TITLE
Handle channels that have empty exercises

### ISSUE THREAD

**Issue by ralphiee22** [learningequality/kolibri#4854]
### Observed behavior

When using a channel that has empty exercises (aka no assessment items), for example, create exam page will still show the channel but when clicking through does not show the subtopics.

### Expected behavior

Don't show channel or show message that says no exercises/assessment items are available for this channel.

### User-facing consequences

Confusion

### Steps to reproduce

Download a db and set table `assessment_metadata`=> field `number_of_assessments`=> 0.

### Context

0.12

**Comment by rtibbles** [comment]
This had previously been fixed here: #3325

Wondering how the regression came in?

**Comment by rtibbles** [comment]
@radinamatic would you be able to confirm that this is still the case?

**Comment by radinamatic** [comment]
I don't think this can be replicated with the current Studio design, as incomplete channel content nodes, such as exercises without questions/assessment items, will not be published. 

I can try to do what was suggested (_Download a db and set table assessment_metadata=> field number_of_assessments=> 0._) to test, I'm just wondering if that's really probable to find in the wild...? 🤔

**Comment by radinamatic** [comment]
Seems that the issue is still extant on 0.14, or at least it does not display the above mentioned _Expected behavior_.

1. I imported a simple channel (fresh facility, no other channels) with 2 topics, one of them containing an exercise with 2 questions. I could see the channel/exercise when creating a quiz.

   ![LEDev21 (start)  Running  - Oracle VM VirtualBox_044](https://user-images.githubusercontent.com/1457929/112076711-3583cb00-8b7b-11eb-8678-742b6f9e400d.png)

2. Stopped the server and manually edited the `number_of_assessments` field in the `content_assessmentmetadata` table to state value 0 instead of 2. Saved the DB and restarted the server. When trying to create a quiz, the channel was still available to select (even though in the DB it technically did not have any _assessment_items_), but when I opened the topic, no exercises were visible, and the **Select all** checkbox was checked:

   ![LEDev21 (start)  Running  - Oracle VM VirtualBox_045](https://user-images.githubusercontent.com/1457929/112077318-89db7a80-8b7c-11eb-907a-64f7bee66f93.png)

I also learned today that the value of `number_of_assessments` field made no difference when creating lessons... 💡
